libs/services/CIR.py

```diff
@@ -44,7 +44,7 @@ def inchikey_to_inchi(self, inchikey):
         args = '{}/stdinchi'.format(inchikey)
```

Comment:
```suggestion
        args = f'{inchikey}/stdinchi'
```

Maybe use *f strings*? I'm not insisting, but it looks more readable to me.

Reply:
Thanks! Most of these are already covered in main from a previous PR, but I still missed some of them. Fixed in ddfad34.